hello_phoenix/firmware/config/target.exs

```diff
@@ -1,5 +1,17 @@
 import Config
 
+config :ui, UiWeb.Endpoint,
+  url: [host: "nerves.local"],
+  http: [port: 80],
+  secret_key_base: "HEY05EB1dFVSu6KykKHuS4rQPQzSHv4F7mGVB/gnDLrIu75wE/ytBXy2TaL3A6RA",
+  live_view: [signing_salt: "AAAABjEyERMkxgDh"],
+  # check_origin: ["//*.local"],
```

Comment:
Should we just set this to `check_origin: false` to simplify possibly errors

Reply:
ah i didn't know that was possible. I think yes.